examples/src/dice_game.cpp

```diff
@@ -19,7 +19,7 @@ int main(int argc, char** argv) {
     double par1 = atof(argv[1]); // number of dice
     double par2 = atof(argv[2]); // number of sides on dice
 
-    std::vector<double> results = simulator({ par1, par2 }, time (NULL) * getpid(), 0, nullptr);
+    std::vector<double> results = simulator({ par1, par2 }, time (NULL) * getpid(), 0);
```

Comment:
```
size_t seed = time (NULL) * getpid();
size_t serial = 0; // this is for your bookkeeping purposes, and shouldn't affect simulator output
```

Reply:
done. i do think there's a worthwhile discussion on the need to pass `serial`.

i understand the use cases, but i think we ought to prefer only *requiring* minimal simulator inputs (basically, parameters) and then flexibly supporting ways to pass more state.

Reply:
have stubbed in an issue for the longer conversation on this.